tests/v2/validation/serviceaccounts/sa_test.go

```diff
@@ -0,0 +1,112 @@
+//go:build (validation || infra.any || cluster.any || sanity) && !stress && !extended
+
+package serviceaccounts
+
+import (
+	"testing"
+	"strings"
+
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/extensions/clusters"
+
+	"github.com/rancher/shepherd/extensions/serviceaccounts"
```

Comment:
Looks like this extension is no longer available and has been moved to actions here: https://github.com/rancher/rancher/blob/main/tests/v2/actions/serviceaccounts/serviceaccounts.go 

Can you change this to: 
"github.com/rancher/rancher/tests/v2/actions/serviceaccounts"